pkg/controller/networkpolicy/validate.go

```diff
@@ -599,20 +599,13 @@ func (v *antreaPolicyValidator) validateAppliedTo(ingress, egress []crdv1beta1.R
 		appliedToEgressRule  = 2
 	)
 
-	appliedToNode := false
 	checkAppliedTo := func(appliedTo []crdv1beta1.AppliedTo, appliedToScope int) (string, bool) {
 		appliedToSvcNum := 0
 		for _, eachAppliedTo := range appliedTo {
 			appliedToFieldsNum := numFieldsSetInStruct(eachAppliedTo)
 			if eachAppliedTo.Group != "" && appliedToFieldsNum > 1 {
 				return "group cannot be set with other peers in appliedTo", false
 			}
-			if eachAppliedTo.NodeSelector != nil {
-				if appliedToFieldsNum > 1 {
-					return "nodeSelector cannot be set with other peers in appliedTo", false
-				}
```

Comment:
I am not sure how removing this check relates to the rest of the changes?

Reply:
My bad. I removed this by mistake.